Application/EdFi.Ods.AdminApi/Program.cs

```diff
@@ -26,11 +26,11 @@
 
 AdminApiVersions.Initialize(app);
 
-//The ordering here is meaningful: Routing -> Auth -> Endpoints
+//The ordering here is meaningful: Logging -> Routing -> Auth -> Endpoints
+app.UseMiddleware<RequestLoggingMiddleware>();
 app.UseRouting();
 app.UseAuthentication();
 app.UseAuthorization();
-app.UseMiddleware<RequestLoggingMiddleware>();
```

Comment:
This was really interesting. I was having inconsistent results between running on my computer (Windows) and running in a container. Moving the request logging to the top of the stack fixed that. And it makes sense to be there: this way, the try / catch in the request logging middleware applies to all subsequent steps in the ASP.Net pipeline, whereas before it was only invoked after the request had already been handled by several other components.